pkgs/development/python-modules/catppuccin-catwalk/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, pillow
+, poetry-core
+,
+}: buildPythonApplication rec {
+  pname = "catppuccin-catwalk";
+  version = "0.4.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "catppuccin_catwalk";
+    hash = "sha256-5TAw5H3soxe9vLhfj1qs8uMr4ybrHlCj4zdsMzvPo6s=";
+  };
+
+  propagatedBuildInputs = [
+    pillow
+  ];
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  # upstream does not have testing
+  doCheck = false;
+
```

```suggestion

  pythonImports = [
    "catwalk"
  ];

```
This is not a library, but `pythonImportsCheck` should provide a simple test.